lms/djangoapps/api_manager/courses/tests.py

```diff
@@ -418,11 +418,11 @@ def test_courses_tree_get(self):
         chapter = response.data['content'][0]
         self.assertEqual(chapter['category'], 'chapter')
         self.assertEqual(chapter['name'], 'Overview')
-        self.assertEqual(len(chapter['children']), 6)
+        self.assertEqual(len(chapter['children']), 5)
```

Comment:
I always get a little nervous when I see test assertions changing -- why are we now only getting five children back from this call?  Is it because of the detached_categories?